analytical_engine/java/grape-jdk/src/main/java/com/alibaba/graphscope/fragment/ArrowFragment.java

```diff
@@ -229,6 +229,9 @@ boolean isOuterVertex(
     <DATA_T> VertexDataColumn<DATA_T> vertexDataColumn(
             int vertexLabelId, int propertyId, @FFISkip DATA_T unused);
 
+    // // @CXXTemplate(cxx = {"uint64_t"}, java = {"Long"})
+    // // @CXXTemplate(cxx = {"double"}, java = {"Double"})
+    // // @CXXTemplate(cxx = {"uint32_t"}, java = {"Integer"})
     // @FFINameAlias("GetData")
```

Comment:
I suggestion to delete those (seems) unused comments.

It looks like this pull request has cleaned many such stuffs, and add some similar things (might be cleaned in further PRs again) at the same time.